lib/index.js

```diff
@@ -59,6 +64,7 @@ internals.Heavy.prototype.start = function () {
             // Retain the same this.load object to keep external references valid
 
             this.load.eventLoopDelay = (this._loadBench.elapsed() - this.settings.sampleInterval);
+            this.load.eventLoopUtilization = PerfHooks.performance.eventLoopUtilization(this._eventLoopUtilization).utilization;
```

Comment:
You need to update `this._eventLoopUtilization`. Otherwise you measure since the start.

Reply:
Fixed.